entries/entry02.md

```diff
@@ -1,8 +1,52 @@
 # Entry 2
```

Comment:
Sources: 1
EDP: 1
Knowledge: 1
Skills: 1
Professionalism: 1

Great job! So excited for your project. Just make sure you prioritize the MVP and then add to it as time allows.